src/main/groovy/org/ods/doc/gen/leva/doc/services/LeVADocumentService.groovy

```diff
@@ -1639,4 +1647,15 @@ class LeVADocumentService {
         junit.getNumberOfTestCases(testData.testResults) - testIssues.count { !it.isUnexecuted }
     }
 
+    private computeRepositoriesDataNeededInDocs(Map data) {
```

Comment:
please, can you change computeRepositoriesDataNeededInDocs by "updateRepositoryData"

Reply:
done